### PR TITLE
Finding the Lost Expedition & Cooking Meat Quest Exile's Reach

### PR DESCRIPTION
**Pull request by ghost**
**Changes proposed:**

-  Script missing quest. All working except removing private spawns on quest drop.

**Tests performed:**

Compiles, tested in game

**Known issues and TODO list:** (add/remove lines as needed)

depends on:
- [x] #29136

### CODE REVIEW

**Comment by malcrom** [comment]
to hell with you nopch

**Comment by acidmanifesto** [comment]
> to hell with you nopch

```
[ 57%] Building CXX object src/server/scripts/CMakeFiles/scripts_events.dir/Events/fireworks_show.cpp.o /home/circleci/project/src/server/scripts/ExilesReach/zone_exiles_reach.cpp:1757:44: fatal error: expected class name struct npc_garrick_summoned_beach : public EscortAI ^ 1 error generated. make[2]: *** [src/server/scripts/CMakeFiles/scripts_exilesreach.dir/build.make:90: src/server/scripts/CMakeFiles/scripts_exilesreach.dir/ExilesReach/zone_exiles_reach.cpp.o] Error 1 
```

**Comment by malcrom** [comment]
Leaving this here to add after personal spawn removal  PR is added.

```
-- Update Summoned data to add quest Finding the Lost Expedition Alliance and Horde

-- Bo Standing, Mithdran Standing, & Lana Jordan Standing
UPDATE `creature_summoned_data` SET `QuestsToDespawnOnQuestRemove` = '59930 59931' WHERE `CreatureID` IN (166787, 166792, 166797);
-- Kee La Standing, Bjorn Stouthands Standing, & Austin Huxworth Standing
UPDATE `creature_summoned_data` SET `QuestsToDespawnOnQuestRemove` = '54951 54952' WHERE `CreatureID` IN (151088, 151089, 154170);
```

**Comment by acidmanifesto** [comment]
All green

File: src/server/scripts/ExilesReach/zone_exiles_reach.cpp [hunk 1]
```diff
@@ -1559,77 +1623,416 @@ struct npc_horde_survivors_beach_q59930_private : public ScriptedAI
...
+
+    void MovementInform(uint32 uiType, uint32 /*uiId*/) override
+    {
+        if (uiType != POINT_MOTION_TYPE)
+            return;
+
+        if (Player* player = ObjectAccessor::GetPlayer(*me, _playerGUID))
+        {
+            player->CastSpell(player, SPELL_UPDATE_PHASE_SHIFT);
+            player->UpdateObjectVisibility();
```
Comment (mdX7) [inline src/server/scripts/ExilesReach/zone_exiles_reach.cpp:1908]: is this required although we are already doing manual phase update via spell?

Comment (malcrom) [reply src/server/scripts/ExilesReach/zone_exiles_reach.cpp:1908]: That is the quest phase shift being updated by spell. And it is required for this quest.

Comment (malcrom) [reply src/server/scripts/ExilesReach/zone_exiles_reach.cpp:1908]: That is required to shift phase on this quest.

**Pull request merged**